### ISSUE TITLE
svcat chokes on azure auth provider

### ISSUE THREAD

**Issue by krancour** [kubernetes-incubator/service-catalog#1818]
I'm going to assume it chokes on all non-default auth providers:

```
$ svcat get classes
Error: No Auth Provider found for name "azure"
```

Relevant section from my kubeconfig:

```
users:
- name: my-cluster-admin
  user:
    auth-provider:
      config:
        access-token: <redacted>
        apiserver-id: <redacted>
        client-id: <redacted>
        expires-in: "3599"
        expires-on: "1520461261"
        refresh-token: <redacted>
        tenant-id: <redacted>
      name: azure
```

**Comment by carolynvs** [comment]
I think this is related to the same problem that we had with the google auth provider. I put in a fix for that in #1695 but maybe that wasn't enough. I'll take a look! 👍

**Comment by krancour** [comment]
D'oh! The problem is I was using an older binary! Sorry!
